CMakeLists.txt

```diff
@@ -66,6 +66,7 @@ jsoncpp_parse_version( ${JSONCPP_VERSION} JSONCPP_VERSION )
 #ENDIF(NOT JSONCPP_VERSION_FOUND)
 SET( JSONCPP_SOVERSION 11 )
 SET( JSONCPP_USE_SECURE_MEMORY "0" CACHE STRING "-D...=1 to use memory-wiping allocator for STL" )
+SET (CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/CMake")
```

Comment:
I'm not sure it's a good idea to override the default ``CMAKE_MODULE_PATH`` value.
You can still include ``cotire.cmake`` relatively to the currently processed ``CmakeLists.txt``:
```
INCLUDE(${CMAKE_CURRENT_SOURCE_DIR}/cotire.cmake)
```